orienteer-core/src/main/java/org/orienteer/core/component/property/PasswordsPanel.java

```diff
@@ -34,6 +33,7 @@ protected void onInitialize() {
 		password.setLabel(labelModel);
 		confirmPassword.add(new AttributeModifier("placeholder", new StringResourceModel("password.placeholder.confirm", labelModel)));
 		confirmPassword.setLabel(new StringResourceModel("password.confirm.label", labelModel));
+		confirmPassword.add(AttributeModifier.append("style", "margin-top: 5px;"));
```

Comment:
Why AttributeModifier needed here? Why just don't update PasswordsPanel.html?

Reply:
Or even in corresponding CSS style?

Reply:
Yes, you are right. I fixed it in 1e09041